CONTRIBUTING.md

```diff
@@ -17,6 +17,7 @@ file.
 * In the project structure dialog, configure an IntelliJ platform SDK
   - point it to the `Contents` directory in your just downloaded copy of IntelliJ Community Edition (e.g, `IntelliJ IDEA CE.app/Contents`)
   - name it `IntelliJ IDEA Community Edition`
+* Optionally, If you want platform sources handy, clone the IntelliJ IDEA Community Edition repo (`git clone https://github.com/JetBrains/intellij-community`) and add a path to your local clone in the `Sourcepaths` tab of your `IntelliJ IDEA Community Edition` SDK and accept all the root folders found by the IDE after scanning.
```

Comment:
Awesome! Thanks for figuring that out. Instead of making this optional I'd prefer to see a comment that you'd get decompiled code if you omit this step. Also, you have to do this for the Dart sources, too?